OrmTester/OrmTesterDesktop/MainWindow.xaml.cs

```diff
@@ -141,10 +141,6 @@ private void GatherCreateManyToManyOperations(TestParametersBuilder builder)
                     {
                         builder.TestBulkCreateManyToMany(repetitions);
```

Comment:
Zmień proszę repetitions na numberOfRecords albo coś w tym stylu, bo trochę to wprowadza w błąd jak się czyta kod